### ISSUE TITLE
Remove QualCoder via deb file

### ISSUE THREAD

**Issue by WPFilmmaker** [ccbogel/QualCoder#15]
Usually when you install a software from deb (once installation is finished) you can the button to remove it as well, which is quite handy and not available for QualCoder. You mentioned it is the first time you make a deb file so maybe you omitted it accidentally? Could you add it?

**Comment by ccbogel** [comment]
Hmm, this actually works (the remove button exists and works) on Mint and Ubuntu. Lubunu must be a bit different, so will look in to it, as in other issue, I will install virtualbox and Lubuntu 16  and try it out.

**Comment by WPFilmmaker** [comment]
Thanks!

**Comment by ccbogel** [comment]
to remove it via the terminal:
sudo apt remove qualcoder

**Comment by ccbogel** [comment]
OK I have installed Lubuntu inside Virtualbox - the version I installed is 16.04.6 (based on the Ubuntu 16.04.6 LTS).I can't get the shared folder to work, but internet access is OK. I uploaded a newer deb file (qualcoder-1.1.deb), this has been checked using lintian when I made it, and does not show errors, only some warnings.
Qualcoder installed successfully when double clicking the deb:
![lubuntu1](https://user-images.githubusercontent.com/5834638/55396754-5d458700-5590-11e9-8cd1-ae94425b546f.png)
It shows up in the Education menu option, even though I classified it as Science:
![lubuntu2](https://user-images.githubusercontent.com/5834638/55396855-95e56080-5590-11e9-953e-0d9476e3524e.png)
Using the Synaptic Package Manager I can mark it for complete removal:
![lubuntu7](https://user-images.githubusercontent.com/5834638/55396969-e8268180-5590-11e9-822c-068d2d4ebddf.png)
 Then press the Apply changes button to remove:
![lubuntu10](https://user-images.githubusercontent.com/5834638/55397097-1efc9780-5591-11e9-837d-108844c847d7.png)
This successfully removes the package. So I will delete the 1.0.deb package as maybe it was not created as correctly as the 1.1.deb package.
